analyzers/src/Directory.Build.targets

```diff
@@ -4,7 +4,9 @@
 
   <PropertyGroup>
     <!-- `BinariesFolder` - binary files folder used by ITs and java packaging -->
-    <BinariesFolder>$(MSBuildThisFileDirectory)\..\packaging\binaries\</BinariesFolder>
+    <BinariesFolder>$(MSBuildThisFileDirectory)..\packaging\binaries\</BinariesFolder>
+    <!-- Avoid SonarAnalyzer.CSharp.Styling to be picked up by Java ITs by putting it outside of the binaries folder -->
```

Comment:
As it's not `outside` anymore, this is enough given the context of this file and the previous comment
```suggestion
    <!-- Avoid SonarAnalyzer.CSharp.Styling to be picked up by Java ITs -->
```